internal/models/tanzukubernetescluster/common/cluster_osimage.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package tkccommon
+
+import (
+	"github.com/go-openapi/swag"
+)
+
+// VmwareTanzuManageV1alpha1ManagementclusterProvisionerTanzukubernetesclusterCommonClusterOSImage The OS image configuration.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.managementcluster.provisioner.tanzukubernetescluster.common.cluster.OSImage
+type VmwareTanzuManageV1alpha1ManagementclusterProvisionerTanzukubernetesclusterCommonClusterOSImage struct {
```

Comment:
I believe this is common and can be reused from other legacy cluster models.

Reply:
I couldn't find it in the existing models. Can you point me to that file?

Reply:
Never mind, that file has other fields too. Let it be this way.